src/components/progressCircular/progress-circular.scss

```diff
@@ -74,7 +75,7 @@ md-progress-circular {
   }
 
 
-  .md-spinner-wrapper.md-mode-indeterminate  {
+  .md-mode-indeterminate .md-spinner-wrapper {
```

Comment:
Ah gotcha! I missed the space between them or I wouldn't have asked (I thought you literally just swapped the order of the selector :smile:).